src/components/container/AIAssistant/AIAssistant.stories.tsx

```diff
@@ -28,6 +28,13 @@ export default {
                 summary: ""
             }
         },
+        showSuggestions: {
+            control: 'boolean',
+            description: 'If turned on the assistant will show suggestions to the user.',
+            defaultValue: {
+                summary: true
+            }
+        },
         appendTools: {
             control: false,
             description: 'If turned on the tools passed in the tools prop will be appended to the default tools.',
```

Comment:
Would be helpful to add a blurb on what the default tools are.

Reply:
This parameter doesn't make too much sense in a story context. Other tools would be passed in here programmatically, and this is where you would say if you wanted the new tools to be the only tools available or whether you want the new tools to be added to the default ones. The default tools are just the list that shows up under the tools parameter here.